### PR TITLE
Remove `@odata.` prefix

### PR DESCRIPTION
**Pull request by HeikoTheissen**

### CODE REVIEW

File: odata-json-format/7 Structural Property.md [hunk 1]
```diff
@@ -409,7 +409,7 @@ OData 4.01 services MUST support both the OData 4.0 representation, for
 requests containing an `OData-Version` header with a value of
 `4.0`, and the OData 4.01 representation, for requests
 containing an `OData-Version` header with a value of `4.01`.
-Clients MUST NOT use `@odata.bind` in requests with an
+Clients MUST NOT use `@bind` in requests with an
```
Comment (ralfhandl) [inline odata-json-format/7 Structural Property.md:412]: In 4.0 services clients MUST use `@odata.bind`, and in 4.01 or later they MUST NOT use it at all. Please revert this change.

File: odata-json-format/7 Structural Property.md [hunk 2]
```diff
@@ -348,7 +348,7 @@ PATCH http://host/service/Products(42) HTTP/1.1
 Content-Type: application/json
 
 {
-  "Category@odata.bind": "Categories(6)"
+  "Category@bind": "Categories(6)"
```
Comment (ralfhandl) [inline odata-json-format/7 Structural Property.md:351]: Please revert, this example is for 4.0 where the `odata.` prefix is required.

File: odata-protocol/11.4 Data Modification.md [hunk 3]
```diff
@@ -212,12 +212,12 @@ entity with links to an existing manager (of managers) and to two existing emplo
...
 {
-  "@odata.type": "#Northwind.Manager",
+  "@type": "#Northwind.Manager",
   "ID": 1,
   "FirstName": "Pat",
   "LastName": "Griswold",
-  "Manager@odata.bind": "http://host/service/Employees(0)",
-  "DirectReports@odata.bind": [
+  "Manager@bind": "http://host/service/Employees(0)",
+  "DirectReports@bind": [
```
Comment (ralfhandl) [inline odata-protocol/11.4 Data Modification.md:220]:
```suggestion
  "DirectReports@odata.bind": [
```

File: odata-protocol/11.4 Data Modification.md [hunk 4]
```diff
@@ -212,12 +212,12 @@ entity with links to an existing manager (of managers) and to two existing emplo
...
 ```json
 {
-  "@odata.type": "#Northwind.Manager",
+  "@type": "#Northwind.Manager",
   "ID": 1,
   "FirstName": "Pat",
   "LastName": "Griswold",
-  "Manager@odata.bind": "http://host/service/Employees(0)",
-  "DirectReports@odata.bind": [
+  "Manager@bind": "http://host/service/Employees(0)",
```
Comment (ralfhandl) [inline odata-protocol/11.4 Data Modification.md:219]:
```suggestion
  "Manager@odata.bind": "http://host/service/Employees(0)",
```

**Review by ralfhandl — CHANGES_REQUESTED** [review]
No `@bind`, has to be `@odata.bind` because it is 4.0 only

**Comment by HeikoTheissen** [comment]
> No `@bind`, has to be `@odata.bind` because it is 4.0 only

So `@bind` is allowed only is 4.0, where it must be `@odata.bind`, right?

**Review by ralfhandl — APPROVED** [review]

**Pull request merged**